app/assets/javascripts/editor.js

```diff
@@ -23,117 +25,107 @@ $E = {
         return code;
       }
     });
-  },
-  setState: function(textarea = 'text-input', preview = 'comment-preview-main', title = 'title') {
-    $E.title = $('#' + title + 'title'); // not sure why this exists? seems like $E.title is always #title
-    $E.textarea = $('#' + textarea);
+  }
+  setState(commentFormID = "main") {
+    $E.textarea = $("#text-input-" + commentFormID);
```

Comment:
`setState` now takes one parameter, the comment form ID. Before, it was taking `preview ID` and `textarea ID`.

Reply:
AWESOME !! more reliable now 🎉